css-penguin/index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link rel="stylesheet" href="./styles.css" />
+    <title>CSS Penguin</title>
```

Comment:
There seems to be some margin collapsing on smaller screens:

![image](https://user-images.githubusercontent.com/2051070/140451292-6486168d-3b36-46e7-b532-90ea088377bc.png)

But setting the viewport meta tag seems to fix this:

```suggestion
    <title>CSS Penguin</title>
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```